src/garage/np/algos/meta_algorithm.py

```diff
@@ -0,0 +1,35 @@
+"""Interface of Meta-RL ALgorithms."""
+import abc
+
+from garage.np.algos import RLAlgorithm
+
+
+class MetaRLAlgorithm(RLAlgorithm):
```

Comment:
can this also host the evaluation calculation code, as in `RLAlgorithm`? these two classes should at least be consistent.

Reply:
I'm not convinced that's actually a great pattern, mostly because then all of the test data needs to get passed into the algorithm, where it could potentially get mixed up with training data. Let's try it being separate for now.